docs/release/release-cycle.md

```diff
@@ -3,7 +3,6 @@
 The release cycle is the time between when we start working on a release on the `main` branch until the `.0` release (e.g. `v1.3.0`) is released.
 
 **Note**: For the `v1.3` and `v1.4` releases we assume each release cycle will last approximately 4 months (~ 17 weeks).
```

Comment:
do we want to add new v1.5 release here as well? Then we have to update the same later at the end of the doc where we refer to  https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-cycle-overview.png. I wonder if this document should be generic and not mention at all release versions

Reply:
I think we should just write that we assume this cycle for 2023 and re-evaluate 2024 (xref: https://github.com/kubernetes-sigs/cluster-api/pull/8148#discussion_r1114103614)

Apart from that we don't have to mention specific releases in my opinion

Reply:
I think the most part of the doc should be very independent of specific release and only mention them as examples.

And then we have the question if we want to maintain the diagram going forward, if we keep it as an example or if we drop it. I think that's a question for the com team (cc @oscr).

Reply:
We discussed this diagram during the comms meeting. We would like to suggest keeping the diagram as is for now. But to also create an improvement task to implement it as code and add it as a task for the release team to update. We feel the diagram is helpful for users and should maybe even be made more prominent. It clearly shows the great effort to keep multiple versions supported and when the end of life is for these versions.

cc @joekr @VibhorChinda @sayantani11

Reply:
Sounds good!

Reply:
@sbueringer do you by chance have a version that is editable? If not we can start to recreate it with other tools.

Reply:
```md
<!-- The release cycle png can be opened and edited in draw.io -->
![Release Cycle Overview](release-cycle-overview.png)
```

Just open the png directly in draw.io :)